### ISSUE TITLE
Errors when trying to install Zonemaster-Engine

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-engine#941]
When trying to install latest version of Zonemaster-Engine from CPAN the error messages below will be outputted. The root cause is unknown, but since the it has been possible to install it, it has probably to do with time stamps in recorded data.

# Work around

If the errors below are the only errors, install with the following commands.

1. Install Zonemaster::LDNS as specified in the installation instruction, but exclude "Zonemaster-Engine" from the command if included (OS dependent).
2. Install Zonemaster::Engine with the following command:
```
sudo cpanm -n Zonemaster::Engine # Linux
cpanm -n Zonemaster::Engine # FreeBSD
```

# Errors outputted
```
t/Test-dnssec.t ........... 362/? 
#   Failed test 'dnssec16-cds-cdnskey-signed.zft-root.rd.nic.fr does not give DS16_CDS_INVALID_RRSIG'
#   at t/Test-dnssec.t line 38.
t/Test-dnssec.t ........... 370/? 
#   Failed test 'dnssec16-cds-cdnskey-unsigned.zft-root.rd.nic.fr does not give DS16_CDS_INVALID_RRSIG'
#   at t/Test-dnssec.t line 38.
t/Test-dnssec.t ........... 426/? 
#   Failed test 'dnssec16-cds-matches-no-dnskey.zft-root.rd.nic.fr does not give DS16_CDS_INVALID_RRSIG'
#   at t/Test-dnssec.t line 38.
t/Test-dnssec.t ........... 434/? 
#   Failed test 'dnssec16-cds-cdnskey-signed.zft-root.rd.nic.fr does not give DS17_CDNSKEY_INVALID_RRSIG'
#   at t/Test-dnssec.t line 38.
t/Test-dnssec.t ........... 442/? 
#   Failed test 'dnssec16-cds-cdnskey-unsigned.zft-root.rd.nic.fr does not give DS17_CDNSKEY_INVALID_RRSIG'
#   at t/Test-dnssec.t line 38.
t/Test-dnssec.t ........... 474/? # Looks like you failed 5 tests of 488.
```

[installation instruction]: https://github.com/zonemaster/zonemaster-engine/blob/master/docs/Installation.md

**Comment by matsduf** [comment]
Will be resolved by #942.

**Comment by matsduf** [comment]
A fix release will soon be provided.

**Comment by matsduf** [comment]
Fixed in v2021.1.2.
